addon/helpers/excerpt.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Why would we want to exclude these files from prettier instead of just letting it run now and being done with it? Same goes for most of the other places this exception was added.

Reply:
It's a good question and I decided to add the answer to the FAQ for lint-to-future 😂  https://github.com/mansona/lint-to-the-future#why-would-you-run-this-instead-of-just-running-lint-fix-to-fix-as-many-problems-as-possible

essentially this PR is too big and we can follow up and fix things in follow-up focused PRs if we would like 👍 but I'm not running prettier on this PR because it's way way too big already

Reply:
Hmm. I'm ok with this if you can commit to it as a fast follow PR. Prettier should really never be disabled, the entire point of it is that we should never have to think about what it's doing.